test/test_jit.py

```diff
@@ -4079,10 +4079,12 @@ def func():
         # generate dicts with built-in types (excluding torch.Tensor)
         xprod = itertools.product(constants, constants)
 
-        def keys_pred(t):
-            return isinstance(eval(t[0]), (list, bool)) or eval(t[0]) is None
+        def is_key_disallowed(t):
```

Comment:
this change was necessary because this code was (in contradiction with the comment above) generating dictionaries with `torch.Tensor` as a key. We explicitly do not expect value comparisons for tensor keys in dictionaries in Python.